recipes/giotto-learn/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: 4710ef554ab4f117fcf3953635a268526e7ea15f57f1e88655ffb425e567e646
+
+build:
+  number: 0
+  skip: true  # [py2k]
+  skip: true  # [win and vs<17]
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pybind11
+    - boost >=1.56.*
+    - setuptools
+    - pip
+    - numpy >=1.17.0
+    - scipy
+    - scikit-learn >=0.21.3
+    - openmp
```

Comment:
I think you still need ``gudhi``

Reply:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=81010&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=833&lineEnd=836&colStart=34&colEnd=27

Reply:
```    /tmp/pip-req-build-m7duy_h5/giotto/externals/bindings/persistent_cohomology_bindings.cpp:8:10: fatal error: gudhi/Persistent_cohomology.h: No such file or directory
     #include <gudhi/Persistent_cohomology.h>
              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    compilation terminated.
```